### PR TITLE
Inspector Icon support.

### PR DESCRIPTION
**Pull request by pq**
Plumbing for widget category icons.

image

This only adds a small handful of (provisional) icons with a TODO to iterate and fill in the rest.  Needless to say, lots of room for bike-shedding. 😄 

@jacob314 @devoncarew

### CODE REVIEW

File: src/io/flutter/inspector/FlutterWidget.java [hunk 1]
```diff
@@ -26,11 +28,64 @@
  */
 public class FlutterWidget {
 
+  enum Category {
+    // TODO(pq): fill in undefined widgets.
+    ACCESSIBILITY("Accessibility", null), ANIMATION_AND_MOTION("Animation and Motion", null),
```
Comment (jacob314) [inline src/io/flutter/inspector/FlutterWidget.java]: Not: is this formatted? I'm a bit surprised there are multiple enum entries per line.

File: src/io/flutter/inspector/DiagnosticsNode.java [hunk 2]
```diff
@@ -38,6 +40,7 @@
   public DiagnosticsNode(JsonObject json, InspectorService inspectorService) {
     this.inspectorService = inspectorService;
     this.json = json;
+    this.widget = FlutterWidget.getCatalog().getWidget(getDescription());
```
Comment (jacob314) [inline src/io/flutter/inspector/DiagnosticsNode.java]: getDescription() isn't the right field to be using for this.
Add TODO(jacobr): provide the runtime type of the value of the node.

Comment (pq) [reply src/io/flutter/inspector/DiagnosticsNode.java]: Done.

File: src/io/flutter/inspector/FlutterWidget.java (hunk 1 above)
Comment (pq) [reply src/io/flutter/inspector/FlutterWidget.java]: Sort of. 😉

**Pull request merged**